CONTRIBUTING.md

```diff
@@ -0,0 +1,115 @@
+# Contributing
+
+Implementing and maintaining a protocol stack is a lot of work, therefore any
+help is appreciated, from creating issues, to contributing documentation, 
+fixing issues and adding new features.
+
+Please follow the best-practice contribution guidelines as mentioned below 
+when submitting any changes.
+
+### Contributions License Agreement
+
+Before contributing, please read the 
+[Contributions License Agreement](./CLA.md).
+Every contributor must sign this agreement before any code is merged into the
+repository.
+
+### Git
+
+Please, do your changes in a dedicated feature-/fix-branch in your forked repo
```

Comment:
Yes you as the "owner" (incl all with direct commit rights) can clone and directly commit to the repo. Then "clone" is right. But all others will have a fork because else they could not commit anything :-)